build-tools/packages/build-cli/package.json

```diff
@@ -43,6 +43,8 @@
 		"build:test:esm": "tsc --project ./src/test/tsconfig.json",
 		"check:biome": "biome check .",
 		"check:format": "npm run check:biome",
+		"check:knip": "knip --no-config-hints",
+		"knip:fix": "knip --fix --fix-type dependencies --no-config-hints",
```

Comment:
`--no-config-hints` -- supresses the hints provided by knip tool to remove the unused `ignoreDependencies` coming from knip.base.config.